examples/with-eip-1193-provider/src/components/auth.tsx

```diff
@@ -0,0 +1,86 @@
+"use client";
+
+import React, { useState } from "react";
+import { Button } from "@/components/ui/button";
+import { UUID } from "crypto";
+import {
+  TurnkeyEIP1193Provider,
+  createEIP1193Provider,
+} from "@turnkey/eip-1193-provider";
+import { signUp } from "@/lib/turnkey";
+import { Email } from "@/lib/types";
+import { getTurnkeyClient } from "@/lib/utils";
+import { sepolia } from "viem/chains";
+import { numberToHex } from "viem";
+import { Input } from "@/components/ui/input";
+
+type AuthProps = {
+  onAuth: (params: {
+    organizationId?: UUID;
+    provider?: TurnkeyEIP1193Provider;
+  }) => void;
+};
+
+const { NEXT_PUBLIC_RPC_URL, NEXT_PUBLIC_ORGANIZATION_ID } = process.env;
+
+const rpcUrl = NEXT_PUBLIC_RPC_URL ?? "";
+const parentOrgId = NEXT_PUBLIC_ORGANIZATION_ID ?? "";
+
+export function Auth({ onAuth }: AuthProps) {
+  const [email, setEmail] = useState<Email | "">("");
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value as Email);
+  };
+
+  const handleLogin = async () => {
+    const turnkeyClient = getTurnkeyClient();
+    console.log({ parentOrgId });
```

Comment:
tiny nit: can remove. or, if we do want to include a log, we can add a bit more context to it